osf/models/sanctions.py

```diff
@@ -560,6 +560,7 @@ class Retraction(EmailApprovableSanction):
 
     initiated_by = models.ForeignKey(settings.AUTH_USER_MODEL, null=True, blank=True)
     justification = models.CharField(max_length=2048, null=True, blank=True)
+    date_retracted = NonNaiveDateTimeField(null=True, blank=True)
```

Comment:
Just FYI on those kwargs' behavior.
![screen shot 2017-04-24 at 2 50 43 pm](https://cloud.githubusercontent.com/assets/3709/25353512/7ddd08d8-28fd-11e7-8b9a-f8769d452e47.png)

via Two Scoops of Django, on someone's desk...